api/python/quilt3/packages.py

```diff
@@ -66,6 +67,9 @@
     MANIFEST_MAX_RECORD_SIZE = DEFAULT_MANIFEST_MAX_RECORD_SIZE
 
 
+HASH_NAME = 'AWSChecksumSHA256'
```

Comment:
What would be a good name for it? (Though if we rely on `TransferConfig`'s chunk sizes, then it's still somewhat AWS-specific)

Reply:
I think we have the right name and prefix in "AWS" because they either calculate it for us or we try to do it their way and that's what matters. If they changed their algorithm we would too. I suppose you could call it "hash of hashes" but they doesn't help future developers who will want to Google "AWS checksum" so I say we leave it as is.

Reply:
> I think we have the right name and prefix in "AWS" because they either calculate it for us or we try to do it their way 

I'd rather say that we define a hash function in a way that it can be efficiently calculated using S3 features in most of our use cases, but in cases when it can't (existing S3 object without calculated `ChecksumSHA256` or with part sizes that are not compliant with our part sizing policy or in-memory data) we calculate hash ourselves.

> If they changed their algorithm we would too.

What this means? That we would produce a different hash for the same data and hash type?